modules/user_accounts/php/edit_user.class.inc

```diff
@@ -1048,7 +1048,7 @@ class Edit_User extends \NDB_Form
                 $password = new \Password($values['Password_hash']);
                 // New password must be different than current one
                 $passwordChanged = \User::factory($this->identifier)
-                    ->passwordChanged($_POST['password']);
+                    ->passwordChanged((string)$password);
```

Comment:
```suggestion
                    ->passwordChanged($_POST['Password_hash']);
```
See discussion for details

Reply:
$password = new \Password($values['Password_hash']); useless??

Reply:
> useless??

No it will throw an InvalidArgumentException when the password is too weak or not long enough:
https://github.com/aces/Loris/blob/a8612ff63289b3dc412241328557a0bca372914e/php/libraries/Password.class.inc#L102

Validate gets called automatically when a Password object is created.

The try-catch beginning on 1047 in `edit_user.php` converts this into a front-end error message if the Password is too weak.

Reply:
Why $_POST['Password_hash']? why not $values['Password_hash'] I can't find any $_POST used in this whole file.

Reply:
`$values['Password_hash']` should work too.